package.json

```diff
@@ -54,7 +54,13 @@
     "Tim Douglas (https://github.com/timdouglas)",
     "(https://github.com/homersimpsons)"
   ],
-  "repository": "Oneflow/handlebars-kit",
+  "repository": {
```

Comment:
I think I read something about the name to be `@oneflow/XXX` too right? can you double-check the list?

Reply:
you are right
![screen3](https://user-images.githubusercontent.com/34857260/132377268-6409ed3e-164f-458e-974b-5d691e789aa2.png)